corehq/messaging/scheduling/models/content.py

```diff
@@ -149,7 +149,7 @@ def send(self, recipient, logged_event, phone_entry=None):
 
         email = Email(
             domain=logged_event.domain,
-            date=datetime.utcnow(),
+            date=logged_subevent.date_last_activity,  # use date from subevent for consistency
```

Comment:
Is this ok because emails and subevents are guaranteed to be 1 to 1, unlike sms?

Reply:
yes, and they don't get updated after being created